code/ui/components/package.json

```diff
@@ -73,7 +73,7 @@
     "@chromaui/addon-visual-tests": "^0.0.77",
     "@popperjs/core": "^2.6.0",
     "@radix-ui/react-scroll-area": "^1.0.5",
-    "@storybook/icons": "^1.1.6",
+    "@storybook/icons": "^1.2.0",
```

Comment:
This needs to be a dependency not a devDependency, otherwise it will get prebundled & therefore duplicated

Reply:
Based on our conversations, I moved the import of @storybook/icons as a devDependency on the manager only. Every other imports are set as dependencies.